### PR TITLE
docs fixes

### PR DESCRIPTION
**Pull request by enkore**
#2450

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -42,13 +42,14 @@ Repository File System
 - A journaling file system is strongly recommended. More information can be
   found in :ref:`file-systems`.
 - |project_name| requires the following file system operations:
+
   - create, open, read, write, seek, close, rename, delete
   - link - when upgrading an Attic repo in-place
   - listdir, stat
-  - posix_fadvise - to not flood the operating system's cache
```
Comment (ThomasWaldmann) [inline docs/installation.rst]: why did you delete this?

**Comment by enkore** [comment]
>  	docs: installation: pre-install considerations, fix crossrefs
>
> posix_fadvise -> only used when available; not part of the filesystem.
